.changes/unreleased/Fixes-20230424-161642.yaml

```
@@ -0,0 +1,7 @@
+kind: Fixes
+body: If --profile specified with dbt-init, create the project with the specified
+  profile
+time: 2023-04-24T16:16:42.994547-04:00
+custom:
+  Author: ezraerb
+  Issue: CT-1418
```

For `changie`, we use the GitHub issue number rather than the Jira issue number.
```suggestion
  Issue: "6154"
```